PowerUp/app/src/main/res/layout/intro_screen_1.xml

```diff
@@ -0,0 +1,5 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    android:layout_height="match_parent" tools:context="powerup.systers.com/IntroActivity2"
+    android:background="@drawable/intro_screen1">
```

Comment:
Feel free to use the screenshots as background images, but add the text programatically. Also, take into account that we work with different devices and orientations, so will probably need to take screenshots for all those and adapt the text placement for each